backend/module/feedback/feedback.go

```diff
@@ -54,13 +54,13 @@ func New(cfg *any.Any, log *zap.Logger, scope tally.Scope) (module.Module, error
 
 // SurveyLookup stores a map of the origin to survey mapping that are
 // provided in the feedback module config
-type SurveyLookup struct {
+type surveyLookup struct {
```

Comment:
just a nit. i think initially I had planned to have this in a helper file so I had it exported